internal/engine/experiment/stunreachability/stunreachability_test.go

```diff
@@ -89,22 +104,23 @@ func TestRunCustomInput(t *testing.T) {
 func TestCancelledContext(t *testing.T) {
 	ctx, cancel := context.WithCancel(context.Background())
 	cancel() // immediately fail everything
-	measurer := stunreachability.NewExperimentMeasurer(stunreachability.Config{})
+	measurer := NewExperimentMeasurer(Config{})
 	measurement := new(model.Measurement)
+	measurement.Input = model.MeasurementTarget(defaultInput)
 	err := measurer.Run(
 		ctx,
 		&mockable.Session{},
 		measurement,
 		model.NewPrinterCallbacks(log.Log),
 	)
-	if err.Error() != "interrupted" {
-		t.Fatal("not the error we expected")
+	if !errors.Is(err, context.Canceled) {
```

Comment:
This line is equivalent to the line above. I'd rather check with `errors.Is` than with strings though.